README.rst

```diff
@@ -92,22 +92,19 @@ My email: amjith.r@gmail.com, Twitter: `@amjithr <http://twitter.com/amjithr>`_
 Detailed Installation Instructions:
 -----------------------------------
 
-OS X:
-=====
+macOS:
```

Comment:
Apple seems to be going back and forth between macOS and OS X. :) Should we maybe say `macOS/OS X`?